src/sage/knots/knotinfo.py

```diff
@@ -1853,25 +1854,24 @@ def link(self, use_item=db.columns().pd_notation, snappy=False):
 
         using ``snappy``::
 
-            sage: K7   = KnotInfo.K7_2
-            sage: k7s  = K7.link(snappy=True); k7s     # optional - snappy
-            <Link: 1 comp; 7 cross>
-            sage: K7.link(K7.items.name, snappy=True)  # optional - snappy
-            <Link 7_2: 1 comp; 7 cross>
-            sage: k7sn = _                             # optional - snappy
-            sage: k7s == k7sn                          # optional - snappy
-            False
-            sage: k7s.sage_link().is_isotopic(k7sn.sage_link()) # optional - snappy
-            True
-
-        but observe::
-
             sage: L2  = KnotInfo.L2a1_1
             sage: l2  = L2.link()
             sage: l2s = L2.link(snappy=True).sage_link()  # optional -  snappy
             sage: l2 == l2s                               # optional -  snappy
+            True
+
+        but observe::
+
+            sage: K7   = KnotInfo.K7_2
+            sage: k7s  = K7.link(snappy=True); k7s        # optional - snappy
+            <Link: 1 comp; 7 cross>
+            sage: K7.link(K7.items.name, snappy=True)     # optional - snappy
+            <Link 7_2: 1 comp; 7 cross>
+            sage: k7sn = _                                # optional - snappy
```

Comment:
```suggestion
            sage: k7sn = K7.link(K7.items.name, snappy=True); k7sn     # optional - snappy
            <Link 7_2: 1 comp; 7 cross>
```
A while-we-are-at-it thing to make this doctest a little bit cleaner.